ash/src/entry.rs

```diff
@@ -29,40 +29,12 @@ pub struct Entry {
 /// Vulkan core 1.0
 #[allow(non_camel_case_types)]
 impl Entry {
-    /// Load entry points from a Vulkan loader linked at compile time
-    ///
-    /// Note that instance/device functions are still fetched via `vkGetInstanceProcAddr` and
-    /// `vkGetDeviceProcAddr` for maximum performance.
-    ///
-    /// ```no_run
-    /// use ash::{vk, Entry};
-    /// # fn main() -> Result<(), Box<dyn std::error::Error>> {
-    /// let entry = Entry::new();
-    /// let app_info = vk::ApplicationInfo {
-    ///     api_version: vk::make_api_version(0, 1, 0, 0),
-    ///     ..Default::default()
-    /// };
-    /// let create_info = vk::InstanceCreateInfo {
-    ///     p_application_info: &app_info,
-    ///     ..Default::default()
-    /// };
-    /// let instance = unsafe { entry.create_instance(&create_info, None)? };
-    /// # Ok(()) }
-    /// ```
-    #[cfg(feature = "linked")]
-    #[cfg_attr(docsrs, doc(cfg(feature = "linked")))]
-    pub fn new() -> Self {
-        // Sound because we're linking to Vulkan, which provides a vkGetInstanceProcAddr that has
-        // defined behavior in this use.
-        unsafe {
-            Self::from_static_fn(vk::StaticFn {
-                get_instance_proc_addr: vkGetInstanceProcAddr,
-            })
-        }
-    }
-
     /// Load default Vulkan library for the current platform
     ///
+    /// Prefer this over [`linked`](Self::linked) when your application can gracefully handle
+    /// environments that lack Vulkan support, and when the build environment might not have the
+    /// Vulkan loader available.
```

Comment:
It might be helpful to rephrase this in such a way that emphasizes the impact on downstream dependencies. 

`Entry::loaded` should be preferred unless you explicitly want to impose a dependency on the Vulkan SDK (e.g. perhaps you depend on validation layers). For example, `bevy` depends on `wgpu`. If `wgpu` decided to used `Entry::linked`, every developer who attempted to experiment with bevy would need to install the Vulkan SDK or face compile time linker errors.

Reply:
Adjusted the wording to make the build environment requirements clearer.